src/pages/iou/steps/MoneyRequestAmountPage.js

```diff
@@ -126,13 +127,14 @@ class MoneyRequestAmountPage extends React.Component {
      * @returns {Object}
      */
     getNewState(prevState, newAmount) {
-        if (!this.validateAmount(newAmount)) {
+        const newAmountWithoutSpaces = this.stripSpacesFromAmount(newAmount);
```

Comment:
Thanks @akinwale, for the PR! Just a small tweak, could you add a description of why we are removing spaces here?